std/json.d

```diff
@@ -1252,7 +1252,8 @@ string toJSON(const ref JSONValue root, in bool pretty = false, in JSONOptions o
                     }
                 }
                 else {
-                    json.put(to!string(val));
+                    import std.format : format;
+                    json.put("%.9g".format(val));
```

Comment:
oops, I thought the internal storage was of type float. 

The problem is that if I use "%.17g", some tests don't pass. "%16.g" is OK.